cypress/integration/recieve-notice/recieve-notice-checkout.spec.js

```diff
@@ -0,0 +1,171 @@
+import moment from 'moment';
+import topMenu from '../../support/fragments/topMenu';
+import settingsMenu from '../../support/fragments/settingsMenu';
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import testTypes from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+
+import SearchPane from '../../support/fragments/circulation-log/searchPane';
+import CirculationRules from '../../support/fragments/circulation/circulation-rules';
+import NoticePolicyApi, { NOTICE_CATEGORIES, NOTICE_ACTIONS } from '../../support/fragments/circulation/notice-policy';
+import NoticePolicyTemplateApi from '../../support/fragments/circulation/notice-policy-template';
+import PatronGroups from '../../support/fragments/settings/users/patronGroups';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+import NewNoticePolicy from '../../support/fragments/circulation/newNoticePolicy';
+import NewNoticePolicyTemplate from '../../support/fragments/circulation/newNoticePolicyTemplate';
+import CheckOutActions from '../../support/fragments/check-out-actions/check-out-actions';
+import DefaultUser from '../../support/fragments/user/defaultUser';
+
+// TODO Add email notice check after checktout: https://issues.folio.org/browse/FAT-185
```

Comment:
check the link. Actually it's not related with adding of email support in our cypress tests

Reply:
omg